ui/pages/confirmations/hooks/useSignatureSecurityAlertResponse.ts

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'react-redux';
```

Comment:
This file should be deprecated with the upcoming alerts work

Reply:
NIT: It will be useful to add this information as comment to the files to be deleted once alerts implementation is there.

Reply:
added @deprecated to warn developers when coming across the hook https://github.com/MetaMask/metamask-extension/pull/24402/commits/b1feedb0ae4a3539bfc99927d3053818fd7b2f0f

I believe it is so far only this file with the related test that would be removed with the alerts implementation in this PR